lua/legendary/data/function.lua

```diff
@@ -43,4 +87,19 @@ function Function:frecency_id()
   return string.format('<function> %s', self.description)
 end
 
+--- Intended to be used by UI filters
+function Function:modeSwitched()
+  return self._mode_switched
+end
+
```

Comment:
Based on the diff it looks like this function isn't needed either.

```suggestion
```

Reply:
It's used in UI display

Reply:
It's used for displaying: https://github.com/mrjones2014/legendary.nvim/pull/415/commits/29513d11cf20d125ad9a26645ab7d31b43db39da#r1366849251

Reply:
Yeah, you don't have to do it that way though -- see the other comments.